images/README.md

```diff
@@ -0,0 +1,66 @@
+# Buildpack v3 reference implementation
+
+## Usage
+
+Create your `workspace` dir:
+
+```sh-session
+$ cd /tmp
+$ mkdir workspace
+$ cp -R /path/to/your/app workspace/app
+```
+
+Create a volume for the cache:
+
+```
+$ docker volume create --name packs_cache
+```
+
+Detect:
+
+```
+$ docker run --rm -v "$(pwd)/workspace:/workspace" packs/build /lifecycle/detector
+```
+
+Analyze:
+
+```
+$ docker run --rm -v "$(pwd)/workspace:/workspace" packs/build /lifecycle/analyzer
```

Comment:
The `packs/util` image has the credential helpers necessary to make this work with private images. Could we use that here instead?